marklogic-data-hub/src/main/java/com/marklogic/hub/cli/AbstractInstallerCommand.java

```diff
@@ -0,0 +1,39 @@
+package com.marklogic.hub.cli;
+
+import com.marklogic.client.ext.helper.LoggingObject;
+import com.marklogic.hub.impl.HubConfigImpl;
+
+import java.io.File;
+import java.util.Map;
+import java.util.Properties;
+
+public abstract class AbstractInstallerCommand extends LoggingObject implements InstallerCommand {
+
```

Comment:
protected HubConfigImpl hubConfig;

public AbstractInstallerCommand(HubConfigImpl hubConfig) {
      this. hubConfig = hubConfig;
}